helmfile/charts/networkpolicy-service/templates/thanos/allow-thanos-receive-distributor.yaml

```diff
@@ -0,0 +1,57 @@
+{{ if .Values.thanos.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: thanos-receive-distributor
+  namespace: thanos
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: receive-distributor
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+      {{- if .Values.global.ingressUsingHostNetwork }}
+        {{- range $ip := .Values.global.scNodes.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      {{- else }}
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: ingress-nginx
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/component: controller
+              app.kubernetes.io/instance: ingress-nginx
+      {{- end }}
+      ports:
+        - port: 19291
+          protocol: TCP
```

Comment:
Missing ingress from Prometheus remote write inside SC.

Reply:
I will take a look at it

Reply:
```suggestion
              app.kubernetes.io/instance: ingress-nginx
      {{- end }}
        - namespaceSelector:
            matchLabels:
              kubernetes.io/metadata.name: monitoring
          podSelector:
            matchLabels:
              app: prometheus
      ports:
        - port: 19291
          protocol: TCP
```